### DISCUSSION TITLE
Looping with GoPro Lab

### DISCUSSION THREAD

**Discussion by hunter-jeremy** [gopro/labs#1043] [General]
Hi,
I'm trying to set up my GoPro as a dashcam. I'm using the GPS with Speed as trigger via the GoPro Lab QR Code. However, I can't determine how to enable looping in this way. If I scan my QR and then change to looping it's not clear if the speed trigger is still active. 

Could you advise if it is possible to configure looping with the speed trigger with GoPro labs?

**Comment by hunter-jeremy** [comment]
I'm trying this blocky, which may work.  
![Blocky](https://github.com/user-attachments/assets/fa5a9416-40bd-4d37-ba60-3806a816e113)

**Reply by dnewman-gpsw** [reply]
looks promising. Consider adding loop points, and end capture speeds.

**Reply by hunter-jeremy** [reply]
Thank you. Do you happen to know if this is the correct command to mute the audio recording? I found two different syntaxes. 
![Screenshot_1](https://github.com/user-attachments/assets/c61e3e97-2e20-48db-b0b6-1809a8be98f7)

**Reply by dnewman-gpsw** [reply]
None of the additional extensions are mixed case. So MUTE is all uppercase.  Use the modern syntax $MUTE=15, oMwxyz is for HERO9 or older.
